frontend/src/views/Search.vue

```diff
@@ -389,6 +426,8 @@ export default class Search extends Vue {
   dateInputStart = this.dateFrom
   dateInputEnd = this.dateFrom
   activeBtn = ''
+  disabledPrevious = false
+  disabledNext = false
```

Comment:
Sen sijaan, että käyttäisit datamuuttujia, voisit käyttää tässä computed propertyjä: https://class-component.vuejs.org/guide/class-component.html#computed-properties
Eli niin, että `disabledPrevious` ja `disabledNext` olisivat funktioita, jotka sisältäisivät logiikan, joka on tällä hetkellä funktiossa `checkIfDateButtonShouldBedisabled`. Tällöin näiden kahden muuttujan tilaa ei tarvitsisi eksplisiittisesti säätää joka kerta kun kun`fetchData`a kutsutaan.